operators/aja_source/metadata.json

```
@@ -0,0 +1,32 @@
+{
+	"operator": {
+		"name": "aja_source",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "0.5.0",
```

`find_package(holoscan)` suggests minimum 1.0?

```suggestion
			"minimum_required_version": "1.0.3",
```